crates/libs/sys/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 
 [package]
 name = "windows-sys"
-version = "0.43.0"
+version = "0.42.1"
```

Comment:
#2166 is almost certainly a breaking change, so while the targets are compatible this won't be.